### ISSUE TITLE
River Guardians Attacking Each Other

### ISSUE THREAD

**Issue by Camawama** [mactso/HarderFishesAndAmphibians#3]
The River Guardians are not too fond of one another. They will sit in the water, killing each other.

**Comment by mactso** [comment]
Yes.  That's by design!  They are like fighting fish and very territorial.

But watch out if it starts raining!  They'll all turn on you at the same time.

Have you fought Slippery Biters yet?
